pymemcache/pool.py

```diff
@@ -72,10 +74,18 @@ def get(self):
                 obj = self._obj_creator()
                 self._used_objs.append(obj)
                 return obj
-            else:
-                obj = self._free_objs.pop()
-                self._used_objs.append(obj)
-                return obj
+            obj = None
+            while self._free_objs:
+                # FIFO try to remove timeout idle objects
+                idle_obj = self._free_objs.popleft()
+                if idle_obj._idle_at + self.obj_idle_timeout > time.time():
```

Comment:
Instead of (re)evaluating `time.time()` each loop iteration, should we snapshot a single time value outside of the loop and consistently refer to it instead?